MapboxCoreNavigation/TunnelIntersectionManager.swift

```diff
@@ -47,31 +47,42 @@ open class TunnelIntersectionManager: NSObject {
     /**
      The flag that indicates whether simulated location manager is initialized.
      */
-    @objc public var isAnimationEnabled: Bool = false
+    @objc var isAnimationEnabled: Bool = false
```

Comment:
@bsudekum Hmm 🤔 is there a reason why we deleted the `public` access level declaration here?

Reply:
@vincethecoder it's not something the developer should be mutating, nor is it something they should be inspecting if they are using the delegate properly.